ansible/index.md

```diff
@@ -1,21 +1,64 @@
 ---
 layout: page
 title: DE Ansible Documentation
-root: ../
 ---
 
-## Using this Repository
+# Table of Contents
+* [Overview](#overview)
+* [Setup](#setup)
+* [Design](#design)
+    * [Directory Layout](#directory-layout)
+    * [Role-Separated Playbooks](#role-separated-playbooks)
+    * [Inventories](#inventories)
+* [Playbooks](#playbooks)
 
+# Overview
+[Ansible](http://www.ansible.com/) is an open source, agentless automation tool. The DE development
+team uses Ansible to provision/update our servers and deploy the DE. However, for this repository,
+we only expose the Ansible scripts that we use for deploying the DE. 
+
+If you intend to use our ansible scripts, we highly suggest that you read the 
+[Ansible documentation](http://docs.ansible.com/ansible/index.html).
+
+# Setup
+
+* [Setup](setup)
+
+# Design
+We have strived to follow Ansible's 
+[best practices](http://docs.ansible.com/ansible/playbooks_best_practices.html). 
+However, we have slightly diverted on the topics of 
```

Comment:
The word, `diverted`, works here, but I would have expected `diverged` to be used.